src/index.js

```diff
@@ -24,6 +24,7 @@ export class BlockSuggestor {
    * @param {number} numBlocksPerCategory the size of each toolbox category
    */
   constructor(numBlocksPerCategory) {
+    this.model = null;
```

Comment:
Include a comment for this to say what it represents

Reply:
Remove initialized to `null` as this is shown in the code line below it. A useful comment might include a note about how it needs to first be set to use `train`, but this isn't necessary.